examples/flutter_gallery/lib/gallery/demos.dart

```diff
@@ -111,6 +113,26 @@ List<GalleryDemo> _buildGalleryDemos() {
       routeName: AnimationDemo.routeName,
       buildRoute: (BuildContext context) => const AnimationDemo(),
     ),
+    GalleryDemo(
+      title: 'Customized Design',
+      subtitle: 'Activity Tracker',
+      icon: GalleryIcons.running,
+      category: _kDemos,
+      routeName: CustomizedDesign.routeName,
+      buildRoute: (BuildContext context) => CustomizedDesign(),
+    ),
+    GalleryDemo(
+      title: 'Welcome Screen',
+      subtitle: 'Watch the playground welcome again',
+      icon: GalleryIcons.refresh,
+      category: _kDemos,
+      routeName: Welcome.routeName,
+      buildRoute: (BuildContext context) => Welcome(
```

Comment:
Prefer `=>` for one-liners that return a value

Reply:
Changed